packages/plugin-vue/src/index.js

```diff
@@ -16,7 +16,7 @@ module.exports = class BugsnagPluginVue {
   }
 
   load (client) {
-    if (this.Vue) {
+    if (this.Vue && this.Vue.config) {
```

Comment:
any particular reason not to check against the vue version? we could even use a switch statement to make supporting new versions easier

Reply:
Well the global Vue.config was removed in v3, but I just thought it made more sense to check if the required API exists rather than check for a particular version.

Really either way would work - and I guess the plugin is already 'versioned' so happy to change to a version check if you think a  would be better.

Reply:
fair point. API over version. 👍🏻